shared/profile/index.js

```diff
@@ -27,67 +35,92 @@ class Profile extends Component<void, Props, void> {
   }
 
   componentDidMount () {
-    this.props.refresh(this.props.username)
+    this.props && this.props.refresh()
   }
 
   componentWillReceiveProps (nextProps) {
-    if (nextProps.username !== this.props.username) {
-      this.props.refresh(nextProps.username)
+    const oldUsername = this.props && this.props.username
+    if (nextProps && nextProps.username !== oldUsername) {
+      nextProps.refresh()
     }
   }
 
   render () {
-    return (
+    return this.props ? (
       <Render
         showComingSoon={!flags.tabProfileEnabled}
         {...this.props}
         proofs={this.props.proofs || []}
         loading={this.props.loading}
```

Comment:
Good spot on the `followers`/`following` below. I think this prop is also covered by the spread above?